gradle.properties

```diff
@@ -12,7 +12,7 @@ org.gradle.jvmargs=-Xmx4g
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
 org.gradle.parallel=true
 
-VERSION_NAME=1.17.8
+VERSION_NAME=1.21.0
```

Comment:
Hmm, why is this change in this PR?

Reply:
This is just for local testing, accidentally checked it in, will revert it.